docs/_snippets/patient-address-response.mdx

```diff
@@ -0,0 +1,26 @@
+  <Expandable title="properties">
+    <ResponseField name="addressLine1" type="string">
+      The address.
+    </ResponseField>
+
+    <ResponseField name="addressLine2" type="string">
+      The address details, for example: `#4451`.
+    </ResponseField>
+
+    <ResponseField name="city" type="string">
+      The city.
+    </ResponseField>
+
+    <ResponseField name="state" type="string">
+      The 2 letter state acronym, for example: `CA`.
+    </ResponseField>
+
+    <ResponseField name="zip" type="string" required>
+      5 digit zip code.
+    </ResponseField>
+
+    <ResponseField name="country" type="string" required>
+      Must be "USA".
```

Comment:
Since this is the response-related file, maybe `Always "USA"` is more accurate?

Reply:
Yeah was actually thinking about this - updated